crates/database/src/orders.rs

```diff
@@ -687,6 +688,25 @@ pub fn solvable_orders(
     sqlx::query_as(OPEN_ORDERS).bind(min_valid_to).fetch(ex)
 }
 
+pub fn open_orders_after<'a>(
+    ex: &'a mut PgConnection,
+    uids: &'a [OrderUid],
+    after_timestamp: DateTime<Utc>,
+) -> BoxStream<'a, Result<FullOrder, sqlx::Error>> {
+    #[rustfmt::skip]
+const OPEN_ORDERS_AFTER: &str = const_format::concatcp!(
+    "SELECT ", ORDERS_SELECT,
+    " FROM ", ORDERS_FROM,
+    " LEFT OUTER JOIN ethflow_orders eth_o on eth_o.uid = o.uid ",
+    " WHERE (o.creation_timestamp > $1 OR o.cancellation_timestamp > $1 OR o.uid = ANY($2))",
```

Comment:
This query returns newly created/cancelled orders and also fetches all the updates from other tables for orders created before, but updated since the last fetch. Didn't really get how this should be changed.

Reply:
> So I guess what's confusing me here is that uid could easily contain an order which was created way before after_timestamp and the function would still return data from that order.

Maybe this can be solved by renaming the function instead of introducing one more SQL query?

Reply:
> 1. fetch all uids of orders created or cancelled after the timestamp
> 2. add those to the list of previously open uids

Or do you mean here to replace with it the SQL query that finds updated UIDs? In that case if we have ~2k orders for each auction, the result will contain x2(~4k) orders. By searching only for the updated UIDs, that amount is expected to decrease.